thirdparty/s3.go

```diff
@@ -43,3 +46,28 @@ func PreSign(r RequestParams) (string, error) {
 	urlStr, err := req.Presign(PresignExpireTime)
 	return urlStr, err
 }
+
+func GetHeadObject(r RequestParams) (*s3.HeadObjectOutput, error) {
+	session, err := session.NewSession(&aws.Config{
+		Region: aws.String(endpoints.UsEast1RegionID),
```

Comment:
Is it guaranteed to be in us-east-1? Seems like users are allowed to specify the region when calling s3.put.

Reply:
thanks for spotting that! done.